lib/spack/spack/cmd/install.py

```diff
@@ -87,6 +88,12 @@ def setup_parser(subparser):
         default=None,
         help="phase to stop after when installing (default None)",
     )
+    subparser.add_argument(
+        "--timer-summary-file",
+        type=str,
+        default=None,
+        help="File to record the top level install timer summary",
```

Comment:
```suggestion
        help=argparse.SUPPRESS,
```

let's not make it public